internal/server/apimigrator_test.go

```diff
@@ -95,13 +96,121 @@ func TestRedirect(t *testing.T) {
 	assert.Assert(t, resp.Result().StatusCode == 200)
 }
 
+func TestRedirectOfRequestAndResponseRewrite(t *testing.T) {
```

Comment:
I don't think this test has the right behaviour. It's setting a redirect from `oldtest` to `test` and also running migrations on `test`. Instead, it should run migrations on `oldtest` since that's the path being modified

Reply:
There's a later test that does it in the order you're expecting. Both should be valid.

Reply:
cool wasn't obvious what it was testing